origin.spec

```diff
@@ -413,6 +418,10 @@ fi
 
 %post node
 %systemd_post %{name}-node.service
+# If accounting is not currently enabled systemd reexec
+if [[ `systemctl show mananager | grep -q -e CPUAccounting=no -e MemoryAccounting=no` == 0 ]]; then
```

Comment:
This is the same as `mananager` basically it's returning the defaults because it's a service that doesn't exist. According to folks #systemd it's not possible to get the true default until v228 where you'd use `systemctl show --global` I don't have a host running v228 or newer so I've not verified that. In absence of a true method to return the default I'd prefer checking the two services we really care about.